packages/blade/src/components/Checkbox/CheckboxIcon/Fade.d.ts

```diff
@@ -0,0 +1,11 @@
+import type { CSSObject } from 'styled-components';
+
+export * from './Fade.web';
+
+export type FadeProps = {
+  show?: boolean;
+  children: React.ReactNode;
+  enter: string;
+  leave: string;
```

Comment:
wouldn't this break on native? since we don't mentioned these props for native and they are required?

Reply:
Fixed.